src/main/webapp/app/entities/modeling-assessment/modeling-assessment.service.ts

```diff
@@ -199,6 +201,20 @@ export class ModelingAssessmentService {
         return assessmentsNames;
     }
 
+    /**
+     * Checks if the feedback text of every feedback item is smaller than the configured maximum length.
+     */
+    isFeedbackTextValid(feedback: Feedback[]): boolean {
+        if (!feedback) {
+            return true;
```

Comment:
So if the feedback is null or undefined, the feedbackText is valid? :thinking:
This seems weird to me, if this is part of the implementation it should be explained why in the JSDoc.

Reply:
It checks if the length of the feedback text is valid for every feedback item in the given feedback list. If there is no feedback, the length of the text in the feedback items is valid. It kind of makes sense to me. I extended the documentation a bit.